src/components/Main/Main.test.js

```diff
@@ -98,14 +98,50 @@ describe('Main', () => {
     })
 
     it('Should load favorites from local storage if the category is favorites', () => {
+      const expected = [{
+        name: 'Luke',
+        species: 'human',
+        favorite: true
+      }]
 
+      localStorage.setItem('favorites', JSON.stringify(expected))
+
```

Comment:
wow, that's awesome that you can just setlocalstorage in testing without mocking. so much simpler than mocking